spec/models/user_spec.rb

```diff
@@ -304,11 +315,11 @@
       let(:invoice_b) { create(:invoice, settled: false) }
       before do
         create(:invoice_product,
-          invoice: invoice_a,
-          user_product: user_with_multiple_products.user_products.first)
+               invoice: invoice_a,
```

Comment:
Use one level of indentation for parameters following the first line of a multi-line method call.